addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -847,6 +860,39 @@ options.Class.include({
             'reset_view_arch': resetViewArch,
         });
     },
+    /**
+     * Caches the rpc.
+     *
+     * @override
+     */
+    async _rpcRequestCached() {
+        const cacheId = JSON.stringify(...arguments);
+        if (!this._rpcCache[cacheId]) {
+            this._rpcCache[cacheId] = this.rpc(...arguments);
+        }
+        return this._rpcCache[cacheId];
+    },
```

Comment:
Not a bad idea but would make more sense to be defined in the base definition of the class in web_editor then IMO. Also remove "Request" from the named IMO.